README.adoc

```diff
@@ -39,22 +39,6 @@ Spring Initializr contains Spring Cloud GCP auto-configuration support through t
 
 Similarly to `GCP Messaging`, `GCP Storage` contains the Google Cloud Storage support with no other dependencies needed.
 
-== Spring Milestones Maven Repository
-
-Currently, the Maven artifacts for the project are only available in the Spring Milestones repository.
-You will want to make sure that the repository is added to your `pom.xml` file or globally in your https://maven.apache.org/settings.html[`settings.xml`] file.
-
-[source,xml]
-----
-<repositories>
-    <repository>
-        <id>spring-milestones</id>
-        <name>Spring Milestones</name>
-        <url>https://repo.spring.io/milestone</url>
-    </repository>
-</repositories>
-----
-
 == Spring Cloud GCP Bill of Materials (BOM)
 
 If you're a Maven user, it's probably a good idea to add our BOM to your maven.xml
```

Comment:
It's not a change from this PR, but this is loose language "probably a good idea".